python/tvm/micro/build.py

```diff
@@ -120,18 +120,35 @@ class AutoTvmModuleLoader:
 
     project_options : dict
         project generation option
+
+    project_dir: str
+        if use_existing is False: The path to save the generated microTVM Project.
+        if use_existing is True: The path to a generated microTVM Project for debugging.
+
+    use_existing: bool
+        skips the project generation and opens transport to the project at the project_dir address.
     """
 
     def __init__(
-        self, template_project_dir: Union[pathlib.Path, str], project_options: dict = None
+        self,
+        template_project_dir: Union[pathlib.Path, str],
+        project_options: dict = None,
+        project_dir: Union[pathlib.Path, str] = None,
```

Comment:
Nit: the [Python docs](https://docs.python.org/3/library/os.html#os.PathLike) suggest using `os.PathLike` for typechecking, which is the abstract base class for `pathlib.Path`.
```suggestion
        template_project_dir: Union[os.PathLike, str],
        project_options: dict = None,
        project_dir: Union[os.PathLike, str] = None,
```